src/components/PatternList.js

```diff
@@ -24,11 +17,19 @@ export const PatternList = ({ navigation }) => {
       onPressHandler={() => navigation.navigate("Lesson")}
     >
       <PatternContainer>
-        {patterns.map((pattern, index) => {
+        {grammars.map((grammar, index) => {
           return (
-            <ListTouchable key={index} onPress={() => navigation.navigate("Lesson")}>
+            <ListTouchable
+              key={index}
+              onPress={() =>
+                navigation.navigate("Lesson", {
+                  description: grammar.description,
```

Comment:
maybe here we should pass in the `grammar` we have have access to `description`, `examples`, and `label` in the `Lesson` component.